Common/src/main/java/com/blamejared/crafttweaker/api/zencode/scriptrun/IScriptLoadingOptionsView.java

```diff
@@ -0,0 +1,30 @@
+package com.blamejared.crafttweaker.api.zencode.scriptrun;
+
+import com.blamejared.crafttweaker.api.ScriptLoadingOptions;
+
+import java.util.Collection;
+import java.util.stream.Stream;
+
+public interface IScriptLoadingOptionsView {
+    
+    boolean isFormat();
+    
+    boolean isExecute();
+    
+    String getLoaderName();
+    
+    Collection<String> getInheritedLoaders();
+    
+    ScriptLoadingOptions.ScriptLoadSource getSource();
+    
+    default Collection<String> getTargetedLoaders() {
+        
+        return Stream.concat(this.getInheritedLoaders().stream(), Stream.of(this.getLoaderName())).toList();
+    }
+    
+    default boolean appliesLoader(final String loader) {
```

Comment:
`appliesForLoader` or `appliesToLoader` ?

Or to these options apply the loader (to something)?